ui/scripts/network.js

```diff
@@ -6504,7 +6432,123 @@
                         }
                     }
                 }
-            }
+            },
+            vpnuser: {
+                        type: 'select',
+                        title: 'VPN Users',
```

Comment:
Would be better to use a new localization key and put the text in messages.properties file. Thanks.